go/vt/vtgate/evalengine/translate.go

```diff
@@ -481,7 +493,7 @@ func translateConvertUsingExpr(expr *sqlparser.ConvertUsingExpr, lookup Translat
 		return nil, err
 	}
 
-	using.Collation, err = translateConvertCharset(expr.Type, lookup)
+	using.Collation, err = translateConvertCharset(expr.Type, false, lookup)
```

Comment:
No `binary` or collation here can be specified, so we can always pass in `false` correctly here.